UI/installer/mp-installer.nsi

```diff
@@ -446,22 +464,25 @@ Section "un.obs-studio Program Files" UninstallSection1
 	RMDir "$INSTDIR\OBS Studio"
 SectionEnd
 
-Section /o "un.User Settings" UninstallSection2
+Section /o "un.Settings, Scenes, etc" UninstallSection2
 	RMDir /r "$APPDATA\obs-studio"
 SectionEnd
 
+# These two don't work for the uninstaller
+# !define MUI_UNCOMPONENTSPAGE_TEXT_DESCRIPTION_INFO "Choose which aspects of OBS you want to uninstall."
+# !define MUI_UNCONFIRMPAGE_TEXT_TOP "Check the compoonents you want to uninstall. Keeping Settings unchecked is recommended."
 !insertmacro MUI_UNFUNCTION_DESCRIPTION_BEGIN
 	!insertmacro MUI_DESCRIPTION_TEXT ${UninstallSection1} "Remove the OBS program files."
-	!insertmacro MUI_DESCRIPTION_TEXT ${UninstallSection2} "Removes all settings, plugins, scenes and sources, profiles, log files and other application data."
+	!insertmacro MUI_DESCRIPTION_TEXT ${UninstallSection2} "Removes all settings, scenes and sources, profiles, log files and other application data.\r\n\r\nCANNOT BE UNDONE."
```

Comment:
Use "log files, and other application data" rather than "log files and other application data".